bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -565,17 +546,13 @@ pub mod pallet {
 				},
 			}
 
-			Self::store_execution_header(
-				update.execution_header.block_hash(),
-				update.execution_header.clone().into(),
-			);
-
 			Ok(())
 		}
 
 		/// Verify that `block_root` is an ancestor of `finalized_block_root` Used to prove that
 		/// an execution header is an ancestor of a finalized header (i.e. the blocks are
 		/// on the same chain).
+		#[allow(dead_code)]
```

Comment:
ditto - I'm assuming this PR is still a WIP - we can't have allow(dead_code) in production code.